DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/board_test/src/CMakeLists.txt

```diff
@@ -22,6 +22,18 @@ if(NOT FPGA_DEVICE MATCHES ".*:.*")
     message(STATUS "This sample is made to target BSPs as this is a benchmarking sample.")
```

Comment:
does it make sense to even support `-DFPGA_DEVICE=Agilex7`? I assumed this design was only really intended for testing BSPs, the default behaviour should be to target the BSP that it was benchmarked on.

Reply:
Will be done in separate PR; TODO: confirm what the default board should be.